src/xcode/ENA/TestPlans/Screenshots.xctestplan

```diff
@@ -43,11 +45,6 @@
         "ENAUITests_04_ExposureSubmissionUITests\/test_screenshot_consentSubmission()",
         "ENAUITests_04_ExposureSubmissionUITests\/test_screenshot_symptomsOnsetDateOption()",
         "ENAUITests_04_ExposureSubmissionUITests\/test_screenshot_symptoms_onset_date_option()",
-        "ENAUITests_04_ExposureSubmissionUITests\/test_screenshot_test_result_negative()",
-        "ENAUITests_04_ExposureSubmissionUITests\/test_screenshot_test_result_pending()",
```

Comment:
are we sure about removing the Screenshots?

Reply:
Seems like a bug of comparing system, I have checked twice the tests are there in the plan and also in the file.